templates/azure/terraform/example/hcl_properties.erb

```diff
@@ -24,7 +27,7 @@
         blocks.each do |block|
 -%>
 
-<%=       prop[0] -%> = {
+<%=       prop[0] -%> <%= "= " if is_tags?(prop)-%>{
```

Comment:
Well, I think for this case, it is not a good choice to introduce that common helper function. Let's do it inline: `<%= '= ' if prop[0] == 'tags' -%>`.